src/engraving/libmscore/tempochangeranged.cpp

```diff
@@ -67,6 +67,20 @@ static const ElementStyle tempoSegmentStyle {
     { Sid::tempoMinDistance, Pid::MIN_DISTANCE }
 };
 
+static const std::unordered_map<TempoChangeType, float> DEFAULT_FACTORS_MAP {
+    { TempoChangeType::Accelerando, 1.25 },
+    { TempoChangeType::Allargando, 0.75 },
+    { TempoChangeType::Calando, 0.5 },
+    { TempoChangeType::Lentando, 0.75 },
+    { TempoChangeType::Morendo, 0.5 },
+    { TempoChangeType::Precipitando, 1.15 },
+    { TempoChangeType::Rallentando, 0.75 },
+    { TempoChangeType::Ritardando, 0.75 },
+    { TempoChangeType::Smorzando, 0.5 },
+    { TempoChangeType::Sostenuto, 0.95 },
+    { TempoChangeType::Stringendo, 1.5 }
```

Comment:
Assuming those flotas are the values for a tempo change, shouldn't the ritardano types have negative values and only the the accelerano type positive ones?

Reply:
Edit: forget about it, they are factors, greater or smaller than 1...